blog/dev_hackathon_v4_release.md

```diff
@@ -0,0 +1,134 @@
+:::
+
+![v4 release social media card](../_static/hackathon_v4_release/v4release_card.png)
+
+The [v4.0 release of the PyMC library](https://twitter.com/pymc_devs/status/1533863780601446401) was officially announced on June 6, 2022.  To achieve this milestone, the PyMC team planned a week-long online sprint beginning May 30, 2022.  The goal was to complete remaining issues on the v4.0 release.  But, *mainly* it was to finalize the documentation and update the website for PyMC.
+
+There are 3 separate websites for [PyMC](https://www.pymc.io/welcome.html):  
+- [Website and Blog](https://www.pymc.io/welcome.html)
+- [Documentation (API)](https://www.pymc.io/projects/docs/en/stable/api.html)
+- [Examples](https://www.pymc.io/projects/examples/en/latest/gallery.html)
+
+And all 3 websites had separate navigation bars and it was confusing for the users, maybe for the library maintainers too?
+
+This Dev sprint was held on the new PyMC Discord server which easily enables sharing video, camera and screen share and provides an easy way to hop between “channels”.  
+
+We shared the sprint events on our new [PyMC Online Meetup](https://www.meetup.com/pymc-online-meetup/events/286202420/) to involve experienced contributors in the community as well. We had a mix of core contributors and newcomers, which is a great combination, as it enables us to be productive, but also to foster the new generation of core developers. The following team members attended, from various locations around the world:  
+
+- Thomas Wiecki (Germany)
+- Ricardo Vieira (Romania)
+- Christian Luhmann (New York, USA)
+- Alex Andorra (France)
+- Michael Osthege (Germany)
+- Oriol Abril Pla (Finland)
+- Chris Fonnesbeck (Nashville, USA)
+- Osvaldo Martin (Argentina)
+- Rob Zinkov (Germany)
+- Reshama Shaikh (New York, USA)
+
+The PyMC project also have begun the 2022 [Google Summer of Code](https://github.com/pymc-devs/pymc/wiki/GSoC-2022-projects) (GSOC) program and the following GSOC folks joined:  
+
+- Connor Hassan (Australia)
+- Purna Chandra Mansingh (India)
+- Danh Phan (Australia)
+- Larry Dong (Canada)
+
+A number of contributors joined from the community:  
+
+- Fernando Irarrázaval
+- Ivan Finaev
+- percevalve
+
+There were lots of people throughout the week, and it is likely we are forgetting some folks. If we forgot you, let us know!
+
+<figure>
+ <img src="../_static/hackathon_v4_release/tweet1_hackathon.png" alt="grid screenshot of 9 people" max-height="20%" /> 
+ <figcaption>
+ Day 2 of PyMC v4.0 Developer Hackathon, June 2, 2022; Photo credit: <a href="https://www.pymc-labs.io/team/thomas-wiecki/">Thomas Wiecki</a>; (from left to right): Alex Andorra, Christian Luhmann, x? , ?, Oriol Abril Pla, Purna C, Ricardo Vieira, ?Rob Zinkov, Thomas Wiecki
+ </figcaption>
+</figure>
```

Comment:
```suggestion
:::{figure-md} hackathon_screenshot

![grid screenshot of 9 people](../_static/hackathon_v4_release/tweet1_hackathon.png)

Day 2 of PyMC v4.0 Developer Hackathon, June 2, 2022; Photo credit: [Thomas Wiecki](https://www.pymc-labs.io/team/thomas-wiecki/); (from left to right): Alex Andorra, Christian Luhmann, Connor Hassan , Maxime Kochurov, Oriol Abril Pla, Purna C, Ricardo Vieira, Rob Zinkov, Thomas Wiecki
:::
```

If needed, we can also add [keyword arguments](https://sphinx-primer.readthedocs.io/en/latest/core/directives.html) to control the image and caption appeareance. AFAIK, both the kwargs accepted by the [figure](https://docutils.sourceforge.io/docs/ref/rst/directives.html#figure) and [image](https://docutils.sourceforge.io/docs/ref/rst/directives.html#image-options) directives are accepted.